ucm2/Intel/sof-essx8336/HiFi.conf

```diff
@@ -77,10 +37,12 @@ SectionDevice."Speaker" {
 
 	EnableSequence [
 		cset "name='Speaker Switch' on"
+		cset "name='Differential Mux' lin1-rin1"
```

Comment:
Changing controls which are evidently assigned to the input for the output device seems very suspicious. I would remove this addition.